docs/getting-started/hello-world.mdx

```diff
@@ -382,16 +353,11 @@ Then build an optimized `.wasm` file:
 
 This will optimize and output a new `hello_soroban.optimized.wasm` file in the same location as the input `.wasm`.
 
-:::tip
-Building optimized contracts is only necessary when deploying to a network with fees or when analyzing and profiling a contract to get it as small as possible. If you're just starting out writing a contract, these steps are not necessary. See [Build] for details on how to build for development.
-:::
-
-[Write a Contract]: #write-a-contract
-[Build]: #build
+:::tip Building optimized contracts is only necessary when deploying to a network with fees or when analyzing and profiling a contract to get it as small as possible. If you're just starting out writing a contract, these steps are not necessary. See [Build] for details on how to build for development. :::
```

Comment:
Does this render correctly? I think the `:::` might need to be on separate lines

```suggestion
:::tip
Building optimized contracts is only necessary when deploying to a network with fees or when analyzing and profiling a contract to get it as small as possible. If you're just starting out writing a contract, these steps are not necessary. See [Build] for details on how to build for development.
:::
```

Reply:
good catch!